app.js

```diff
@@ -1,16 +0,0 @@
-const express = require("express")
-const app = express() ;
-const bodyParser = require("body-parser")
-const appRoutes = require("./router/routes")
-require('dotenv').config() ; 
-
-const PORT =  process.env.PORT || 7000 ;
```

Comment:
Do not use 7000 directly, declare it as a variable 

const DEFAULT_PORT = 7000